classic/helpers.go

```diff
@@ -0,0 +1,78 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package classic
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/ubuntu-core/snappy/dirs"
+)
+
+func mountpoint(path string) bool {
+	err := exec.Command("mountpoint", path).Run()
+	// man-page: zero if the directory is a mountpoint, non-zero if not
+	return err == nil
+}
+
+func bindmount(src, dstPath, remountArg string) error {
+	dst := filepath.Join(dirs.ClassicDir, dstPath)
+	// already mounted
```

Comment:
The reason we need a comment here is likely because the function name isn't more clear by itself. `isMounted` or something similar might make it redundant, maybe.